ui/app/rif/pages/domainsDetailPage/domainDetailActive/addNewTokenNetworkAddress/addNewChainAddressToResolver.js

```diff
@@ -22,23 +23,22 @@ class AddNewChainAddressToResolver extends Component {
     };
 	}
   updateChainAddress = (selectedOption) => {
-    console.log('SELECTED OPTION', selectedOption);
 		this.setState({ selectedChainAddress: selectedOption });
-		this.props.updateChainAddress(selectedOption.chain);
+    this.props.updateChainAddress(selectedOption.chain);
 	}
 	updateAddress = (e) => {
 		this.setState({ insertedAddress: e.target.value });
-		this.props.updateAddress(e.target.value);
+    this.props.updateAddress(e.target.value);
 	}
 
 	render () {
     const selectValue = ({value}) => {
       const icon = value.icon ? value.icon : DEFAULT_ICON;
       return (
-        <div className={'add-new-multicrypto-select-value'}>
+        <div className={''}>
```

Comment:
Why is this class removed and why we have an empty string now, empty string it's the same as to not use the className attribute

Reply:
This will be empty, cause @fmelo needs to put some styling here

Reply:
I deleted all classnames empty